lib/reports/subscriber_lists_report.rb

```diff
@@ -0,0 +1,70 @@
+class Reports::SubscriberListsReport
+  attr_reader :start_of_day, :end_of_day
+
+  CSV_HEADERS = %i[title
+                   slug
+                   matching_criteria
+                   created_at
+                   individual_subscribers
+                   daily_subscribers
+                   weekly_subscribers
+                   individual_updates_for_date
+                   unsubscribed].freeze
+
+  def initialize(date)
+    @start_of_day = Time.zone.parse(date)
+    @end_of_day = Time.zone.parse(date).end_of_day
```

Comment:
We should validate this is in the past.

Reply:
Also, given our new retention policy, we should error if it's over a year ago.

Reply:
Good point!